run/VarTierDiseaseDBFalse.R

```diff
@@ -139,6 +138,7 @@ anno$TierAD[anno$Gene %in% GeneTier4] = anno$TierAR[anno$Gene %in% GeneTier4] <-
 # find the non-Coding Var to keep
 nonCodingVar2keep <- c()
 
+# It never executed 
```

Comment:
I wonder if it occurs an error when this branch is executed.

If so, can we get input from original author if it's okay to remove the lines?

Reply:
are you taking about removing `nonCodingVar2Keep`?

Reply:
nvm, I misunderstood, it is okay to remove, I think.